route.scala

```diff
@@ -207,6 +208,21 @@ package object route {
             case applyType: Term.ApplyType =>
               Param(extractParamTerm(applyType, false, aliasDesc))
           }
+        case Term.ApplyType(Term.Name("params"), Seq(Type.Name(typeName))) =>
```

Comment:
It may be, but quasiquotes aren't used anywhere else; I constructed the patterns by printing out failed matches in tests and it was more natural and convenient to just copy-paste them into the code - and it's clearer that they mean exactly what I want, being extracted from use-case examples.